lnwire/update_add_htlc.go

```diff
@@ -92,32 +98,80 @@ var _ Message = (*UpdateAddHTLC)(nil)
 //
 // This is part of the lnwire.Message interface.
 func (c *UpdateAddHTLC) Decode(r io.Reader, pver uint32) error {
+	// msgExtraData is a temporary variable used to read the message extra
+	// data field from the reader.
+	var msgExtraData ExtraOpaqueData
+
 	if err := ReadElements(r,
 		&c.ChanID,
 		&c.ID,
 		&c.Amount,
 		c.PaymentHash[:],
 		&c.Expiry,
 		c.OnionBlob[:],
-		&c.ExtraData,
+		&msgExtraData,
 	); err != nil {
 		return err
 	}
 
+	// Extract TLV records from the extra data field.
 	blindingRecord := c.BlindingPoint.Zero()
-	tlvMap, err := c.ExtraData.ExtractRecords(&blindingRecord)
+
+	extraDataTlvMap, err := msgExtraData.ExtractRecords(&blindingRecord)
 	if err != nil {
 		return err
 	}
 
-	if val, ok := tlvMap[c.BlindingPoint.TlvType()]; ok && val == nil {
+	if val, ok := extraDataTlvMap[c.BlindingPoint.TlvType()]; ok && val == nil { //nolint:lll
```

Comment:
nit: add the `//nolint` comment _above_ the line to prevent it becoming even longer?

Reply:
Or split the assignment and the condition into separate statements/lines.